source/melpomene/src/cli.rs

```diff
@@ -0,0 +1,20 @@
+use crate::{sim_drivers::tcp_serial, sim_tracing};
+use clap::Parser;
+use std::net::SocketAddr;
+
+#[derive(Parser, Debug)]
+#[clap(author, version, about)]
```

Comment:
Dunno if it suits the way this project likes things, but I tend to use the git-version crate and `version=git_version!()` on the clap struct.